Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,27 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/zipball/master"
+  version "1.34.1"
+  sha256 "775430498a8a38d6bce59e265b948ff47fafb5e00aea3fc5d3c47adcd650a36c"
+
+  depends_on "maven" => :build
+
+  def install
+    libexec.install Dir['*']
+    cd libexec
+
+    system "mvn clean install"
+
+    (bin / "gcviewer.sh").write <<-EOS.undent
+      #!/bin/sh
+
+      java -jar #{libexec}/target/gcviewer-#{version}.jar
+    EOS
+
+    bin.install_symlink "#{bin}/gcviewer.sh" => "gcviewer"
+  end
+
+  test do
+    assert_equal "#{libexec}/target/gcviewer-#{version}.jar", `ls #{libexec}/target/gcviewer-#{version}.jar 2>&1`.strip
```

Comment:
Does it have a ` --version ` or ` --help ` CLI interface or anything? Or could we fake a gc log and use that?

If it doesn't and we can't, just bin the test entirely. It'll cause the bot to sulk that it doesn't have a test, but the maintainers can ignore the bot at their leisure. There's plenty of Java stuff that can't be tested in the core, so this won't be alone if it's necessary.